### PR TITLE
Added all distro logs to Universal NRP workflow and updated AzureLinux insiders-fast enlistment instructions

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

* Included all distro logs for UniversalNRP test workflow
* Updated enlistment instructions for `insiders-fast` repository

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

**Review by MariusNi — DISMISSED** [review]
:shipit:

File: .github/workflows/universalnrp-test-run.yml [hunk 1]
```diff
@@ -112,5 +112,5 @@ jobs:
       - uses: actions/upload-artifact@v3
         if: success() || failure()
         with:
-          name: universal-nrp-test-${{ inputs.install-osconfig == true && 'withOSConfig' || 'noOSConfig' }}-logs
+          name: universal-nrp-test-${{ inputs.target }}-${{ inputs.install-osconfig == true && 'withOSConfig' || 'noOSConfig' }}-logs
```
Comment (robertschaedler3) [inline .github/workflows/universalnrp-test-run.yml:115]: nit: noOSConfig -> withoutOSConfig

**Review by robertschaedler3 — APPROVED** [review]

File: .github/workflows/universalnrp-test-run.yml (hunk 1 above)
Comment (AhmedBM) [reply .github/workflows/universalnrp-test-run.yml:115]: yeah that sounds better, will address

**Review by MariusNi — APPROVED** [review]
:shipit:

**Pull request merged**